### PR TITLE
feat: add report a bug in user dropdown

### LINKED ISSUE #206: Add ability to report bugs from the user dropdown
There should be a link to "report a bug" that links to https://github.com/open-sauced/hot/issues/new?assignees=&labels=%F0%9F%91%80+needs+triage%2C%F0%9F%90%9B+bug&template=bug_report.yml&title=Bug%3A+

Image

### PR DESCRIPTION
**Pull request by ddsuhaimi**
## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
fixes #206 

## Mobile & Desktop Screenshots/Recordings
image

## Added tests?

- [ ] 👍 yes
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [X] 🙅 no documentation needed

### CODE REVIEW

File: src/components/PrimaryNav.tsx [hunk 1]
```diff
@@ -96,6 +97,21 @@ const PrimaryNav = (): JSX.Element => {
                   )}
                 </Menu.Item>
 
+                <Menu.Item>
+                  {({active}) => (
+                    <button
+                      onClick={async () => {
+                        window.open(bugReportLink);
```
Comment (bdougie) [inline src/components/PrimaryNav.tsx:104]: <a href={bugReportLink} />
  <button></button>
</a>

interesting use of window.open. Curious if there is value over using a href?

Comment (bdougie) [reply src/components/PrimaryNav.tsx:104]: I suppose less lines and use of button onClick may be more accesible?

**Review by bdougie — CHANGES_REQUESTED** [review]
Per [this s/o](https://stackoverflow.com/questions/1667416/window-location-href-vs-clicking-on-an-anchor/1667512#1667512) answer, could you swap this to href?

**Comment by ddsuhaimi** [comment]
@bdougie no problem

**Comment by ddsuhaimi** [comment]
@bdougie I have updated the PR, lemme know if you need me to change anything else

**Review by 0-vortex — APPROVED** [review]
LGTM!

**Review by 0-vortex — APPROVED** [review]

**Pull request merged**